config/storage.yml

```diff
@@ -9,10 +9,10 @@ local:
 # Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
 # amazon:
```

Comment:
@josephelias-rubyians  Rails uses master.key and credentials.yml file for storing ENV. Are you using them?